libs/shared/profile-form/src/lib/shared-profile-form.tsx

```diff
@@ -0,0 +1,258 @@
+import React from 'react';
+import FormControl from '@material-ui/core/FormControl/FormControl';
+import TextField from '@material-ui/core/TextField/TextField';
+import Button from '@material-ui/core/Button';
+
+import './shared-profile-form.scss';
+import Grid from '@material-ui/core/Grid/Grid';
+import Box from '@material-ui/core/Box/Box';
+import { useForm, Controller } from 'react-hook-form';
+import Select from '@material-ui/core/Select/Select';
+import Input from '@material-ui/core/Input/Input';
+import Chip from '@material-ui/core/Chip/Chip';
+import MenuItem from '@material-ui/core/MenuItem';
+import InputLabel from '@material-ui/core/InputLabel';
+
+/* eslint-disable-next-line */
+interface data {
+  id: string;
+  name: string;
+}
+/* eslint-disable-next-line */
+export interface SharedProfileFormProps {
+  onSubmit: (submitData: {
+    description: string;
+    firstName: string;
+    lastName: string;
+    photo: FileList;
+    fieldOfProfession: string;
+    company: string;
+    categories: string[];
+    city: string;
+    country: string;
+    phone: string;
+    web: string;
+  }) => void;
+  categories: data[];
+  cities: data[];
+  countries: data[];
+}
+
+export const SharedProfileForm = (props: SharedProfileFormProps) => {
+  const { handleSubmit, register, control, errors } = useForm();
+
+  const [categories, setCategories] = React.useState<string[]>([]);
+  const [city, setCity] = React.useState<string>('');
+  const [country, setCountry] = React.useState<string>('');
+
+  const handleChange = (event: React.ChangeEvent<{ value: unknown }>) => {
```

Comment:
don't use unknown and allso remove React. and ChangeEvent import same as useState